build.xml

```diff
@@ -2037,6 +2043,12 @@
         </classpath>
     </taskdef>
     <mkdir dir=".settings" />
+    <echo file=".settings/org.eclipse.jdt.core.prefs"><![CDATA[eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.ignoreUnnamedModuleForSplitPackage=enabled
```

Comment:
Needed to get around `high-scale-lib` possessing java.util and java.util.concurrent packages.